src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/SymbolMatcherTests.cs

```diff
@@ -1847,6 +1847,69 @@ static void Main()
             Assert.Equal("<>9__0_0", field1.Name);
         }
 
+        [Fact]
+        public void SynthesizedDelegates_03()
+        {
+            // This test checks that lambdas between source0 and source1 are not the same,
+            // as they differ in default parameter value or `params` modifier.
+            var source0 = """
+                class C
+                {
+                    unsafe static void F()
+                    {
+                        var _1 = (int a, int b = 1) => a;
+                        var _2 = (int* x, int[] c) => c;
+                    }
+                }
+                """;
+            var source1 = """
+                class C
+                {
+                    unsafe static void F()
+                    {
+                        var _1 = (int a, int b = 2) => a;
```

Comment:
>int b = 2 [](http://example.com/codeflow?start=41&length=9)

I think it would be good to cover scenario no default value vs. default value.

Reply:
Thanks, added.